### DISCUSSION TITLE
selection line move up/down is exist?

### DISCUSSION THREAD

**Discussion by TinyProbe** [helix-editor/helix#5025] [Q&A]
If not, help me.

```toml
C-j = [
"extend_to_line_bounds", "delete_selection", "paste_after",
"move_line_down", "select_mode", "goto_last_modification",
"move_char_left", ]
C-k = [
"extend_to_line_bounds", "delete_selection", "move_line_up",
"move_line_up", "paste_after", "move_line_down",
"select_mode", "goto_last_modification", "move_char_left", ]
```

This binding would be work like selection line up/down.
It works when I run it one by one, but it doesn't work when I press it with a hotkey.

what should i do... ?

**Comment by the-mikedavis** [comment]
See the discussions and example configs in https://github.com/helix-editor/helix/issues/2245 and https://github.com/helix-editor/helix/pull/4545

**Reply by TinyProbe** [reply]
oh yeah~!
